eng/Versions.props

```diff
@@ -68,7 +68,6 @@
     <MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>14.3.25407-alpha</MicrosoftInternalVisualStudioShellInterop140DesignTimeVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta1-63011-01</MicrosoftMetadataVisualizerVersion>
     <MicrosoftMSXMLVersion>8.0.0.0-alpha</MicrosoftMSXMLVersion>
-    <MicrosoftNetCompilersToolsetVersion>3.2.0-beta4-19312-15</MicrosoftNetCompilersToolsetVersion>
```

Comment:
So the point is, let's use the same compiler to build Roslyn as we give to everyone else via Arcade?

Reply:
Exactly.